gradle.properties

```diff
@@ -20,6 +20,9 @@ android.disableAutomaticComponentCreation=true
 # Signals to our own plugin that we are building within the repo.
 app.cash.redwood.internal=true
 
+SONATYPE_HOST=DEFAULT
+RELEASE_SIGNING_ENABLED=true
```

Comment:
This only attempts to sign the artifacts when we start publishing to maven central.